app/src/org/commcare/android/models/AsyncNodeEntityFactory.java

```diff
@@ -53,7 +53,7 @@ public AsyncNodeEntityFactory(Detail d, EvaluationContext ec) {
     }
 
     @Override
-    public Entity<TreeReference> getEntity(TreeReference data) throws SessionUnavailableException {
```

Comment:
Don't understand why 'throws SessionUnavailableException' was here. Taking it out.